src/main/kotlin/com/cognifide/gradle/aem/base/vlt/VltRunner.kt

```diff
@@ -48,11 +48,29 @@ class VltRunner(val project: Project) {
             logger.info("JCR content directory to be checked out does not exist: ${contentDir.absolutePath}")
         }
 
+        checkoutFilter.rootPaths.forEach { createSiblingCpyFiles(File(workingDir, it)) }
```

Comment:
this is for cleaning purposes and should not change `aemCheckout` directly. it is has matter only when `aemClean` is being executed immediatelly after so that I think it should be implemented in a different way


within CleanTask 

```
def checkoutTask = project.gradle.taskGraph.getTaskByName('aemCheckout')
if (checkoutTask != null) // is being executed {
checkoutTask.doFirsft { checkoutFilter.rootPaths.forEach { createSiblingCpyFiles(File(workingDir, it)) } }
}
```

```